DESCRIPTION.md

```diff
@@ -8,11 +8,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v3.12.2(TBD)
-  - Enhanced error handling for asynchronous queries, providing more detailed and informative error messages when an async query fails.
-  - Improved implementation of `snowflake.connector.util_text.random_string` to avoid collisions.
-  - If the account specifies a region, and that region is in China, the TLD is now inferred to be snowflakecomputing.cn.
-  - Changed loglevel to WARNING for OCSP fail-open warning messages (was: ERROR)
+- v3.12.2(September 11,2024)
+  - Improved error handling for asynchronous queries, providing more detailed and informative error messages when an async query fails.
+  - Improved inference of TLD for accounts specifying a region in China, now defaulting to snowflakecomputing.cn.
```

Comment:
is TLD a well-understood thing?

Reply:
I debreivated it to be "top level domain"

Reply:
In that case, grammatically, it should be top level domains